src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +29,8 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.debug("getAllProductsFromShoppingCart finished work successful."
```

Comment:
debug level is used when indicating that some method was called or in other debugging issues
info level is used to track business logic. here it is important for business logic that products were fetched from DB
see TODO
```suggestion
        logger.info("Successfully fetched data from DB"
```